### ISSUE TITLE
Do you have the code for a single CPU?

### ISSUE THREAD

**Issue by werweassd** [shachoi/RobustNet#16]
Thank you so much for your great work! 
I am very interested in your work and am trying your code. 
However, I only have one gpu. After modifying the code and training the model, it is found that the baseline performance has dropped significantly.
Do you have the code for a single CPU?
 I would be very grateful if you could provide it to me.

**Comment by shachoi** [comment]
The smallest model we tested is ShuffleNet. Our training script requires 16 batch size over 2 GPUs (8batch X 2GPUs). So you should modify the script as follows: --nproc_per_node=1 / --bs_mult 16 \
If the requirements of this training script exceed the memory of your GPU, there is no other way than to use a model smaller than ShuffleNet. Reducing the batch size will cause performance degradation.

**Comment by werweassd** [comment]
Dear author, thank you so much for your effort!
Best regards,
